pillarbox-core-business/src/main/java/ch/srgssr/pillarbox/core/business/integrationlayer/service/Vector.kt

```diff
@@ -6,25 +6,29 @@ package ch.srgssr.pillarbox.core.business.integrationlayer.service
 
 import android.content.Context
 import android.content.res.Configuration
+import ch.srgssr.pillarbox.core.business.integrationlayer.service.Vector.MOBILE
+import ch.srgssr.pillarbox.core.business.integrationlayer.service.Vector.TV
 
 /**
- * Vector
+ * Provides constants and utilities to determine the playback vector ([MOBILE] or [TV]).
```

Comment:
It is not really a playback more device.

Reply:
Good catch! I'll update it